<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>624</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QSplitter" name="ui_splitter_A">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="layoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="ui_gBox_view">
         <property name="title">
          <string>View choices</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="ui_chk_discs">
            <property name="text">
             <string>Discs</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ui_chk_ducts">
            <property name="text">
             <string>Oil ducts</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="ui_chk_pressboards">
            <property name="text">
             <string>Pressboards</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="ui_gBox_coilGeom">
         <property name="title">
          <string>Coil geometry</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Radial spacers
width</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Number of
radial spacers</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="ui_le_radSpacerWidth"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Coil height</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Coil width</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="ui_le_coilHeight"/>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="ui_le_coilWidth"/>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="ui_spBox_nbre_radSpacers"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="ui_le_innerRadius"/>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="ui_lbl_innerRadius">
            <property name="text">
             <string>Inner radius</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="ui_spBox_nbreDiscs"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Number of discs</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QSplitter" name="ui_splitter_B">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="ui_splitter_C">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QFrame" name="ui_frame_viewAll">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
       <widget class="QFrame" name="ui_frame_legend">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Ctrl + mouse wheel: zoom and geometry drag
Mouse click, Crtrl, Shift, Ctrl+A: items selection</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ui_btn_symTB">
            <property name="text">
             <string>Symmetry top -&gt; bottom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="ui_btn_symBT">
            <property name="text">
             <string>Symmetry bottom -&gt; top</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QFrame" name="ui_frame_viewZoom">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>ui_chk_discs</tabstop>
  <tabstop>ui_chk_ducts</tabstop>
  <tabstop>ui_chk_pressboards</tabstop>
  <tabstop>ui_spBox_nbreDiscs</tabstop>
  <tabstop>ui_spBox_nbre_radSpacers</tabstop>
  <tabstop>ui_le_radSpacerWidth</tabstop>
  <tabstop>ui_le_innerRadius</tabstop>
  <tabstop>ui_le_coilHeight</tabstop>
  <tabstop>ui_le_coilWidth</tabstop>
  <tabstop>ui_btn_symTB</tabstop>
  <tabstop>ui_btn_symBT</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
